### PR TITLE
Update Portuguese localization

### PR DESCRIPTION
**Pull request by luan-u**

### CODE REVIEW

File: src/locale/locales/pt-BR/messages.po [hunk 1]
```diff
@@ -232,91 +232,91 @@ msgstr "{estimatedTimeMins, plural, one {minuto} other {minutos}}"
...
 
 #: src/view/com/notifications/FeedItem.tsx:338
 msgid "{firstAuthorLink} liked your custom feed"
-msgstr ""
+msgstr "{firstAuthorLink} curtiu seu feed personalizado"
 
 #: src/view/com/notifications/FeedItem.tsx:234
 msgid "{firstAuthorLink} liked your post"
-msgstr ""
+msgstr "{firstAuthorLink} curtiu sua postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:267]:
```suggestion
msgstr "{firstAuthorLink} curtiu seu post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 2]
```diff
@@ -232,91 +232,91 @@ msgstr "{estimatedTimeMins, plural, one {minuto} other {minutos}}"
...
 
 #: src/view/com/notifications/FeedItem.tsx:258
 msgid "{firstAuthorLink} reposted your post"
-msgstr ""
+msgstr "{firstAuthorLink} repostou sua postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:271]:
```suggestion
msgstr "{firstAuthorLink} repostou seu post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 3]
```diff
@@ -232,91 +232,91 @@ msgstr "{estimatedTimeMins, plural, one {minuto} other {minutos}}"
...
 
 #: src/view/com/notifications/FeedItem.tsx:215
 msgid "{firstAuthorName} and {additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}} liked your post"
-msgstr ""
+msgstr "{firstAuthorName} e {additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}} curtiram sua postagem"
 
 #: src/view/com/notifications/FeedItem.tsx:239
 msgid "{firstAuthorName} and {additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}} reposted your post"
-msgstr ""
+msgstr "{firstAuthorName} e {additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}} repostaram sua postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:291]:
```suggestion
msgstr "{firstAuthorName} e {additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}} repostaram seu post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 4]
```diff
@@ -232,91 +232,91 @@ msgstr "{estimatedTimeMins, plural, one {minuto} other {minutos}}"
...
 
 #: src/view/com/notifications/FeedItem.tsx:324
 msgid "{firstAuthorName} liked your custom feed"
-msgstr ""
+msgstr "{firstAuthorName} curtiu seu feed personalizado"
 
 #: src/view/com/notifications/FeedItem.tsx:220
 msgid "{firstAuthorName} liked your post"
-msgstr ""
+msgstr "{firstAuthorName} curtiu sua postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:311]:
```suggestion
msgstr "{firstAuthorName} curtiu seu post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 5]
```diff
@@ -232,91 +232,91 @@ msgstr "{estimatedTimeMins, plural, one {minuto} other {minutos}}"
...
 
 #: src/view/com/notifications/FeedItem.tsx:244
 msgid "{firstAuthorName} reposted your post"
-msgstr ""
+msgstr "{firstAuthorName} repostou sua postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:315]:
```suggestion
msgstr "{firstAuthorName} repostou seu post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 6]
```diff
@@ -598,15 +598,15 @@ msgstr "Adicionar texto alternativo (opcional)"
 #: src/screens/Settings/Settings.tsx:364
 #: src/screens/Settings/Settings.tsx:367
 msgid "Add another account"
-msgstr ""
+msgstr "Adicionar outra conta"
 
 #: src/view/com/composer/Composer.tsx:713
 msgid "Add another post"
-msgstr ""
+msgstr "Adicionar outra postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:605]:
```suggestion
msgstr "Adicionar outro post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 7]
```diff
@@ -632,7 +632,7 @@ msgstr "Adicionar palavras/tags silenciadas"
 
 #: src/view/com/composer/Composer.tsx:1228
 msgid "Add new post"
-msgstr ""
+msgstr "Adicionar nova postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:635]:
```suggestion
msgstr "Adicionar novo post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 8]
```diff
@@ -1057,7 +1057,7 @@ msgstr "Tem certeza que deseja descartar este rascunho?"
 
 #: src/view/com/composer/Composer.tsx:828
 msgid "Are you sure you'd like to discard this post?"
-msgstr ""
+msgstr "Tem certeza de que deseja descartar esta postagem?"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:1060]:
```suggestion
msgstr "Tem certeza de que deseja descartar este post?"
```

File: src/locale/locales/pt-BR/messages.po [hunk 9]
```diff
@@ -1120,21 +1120,21 @@ msgstr "Voltar"
 #: src/view/screens/Lists.tsx:104
 #: src/view/screens/ModerationModlists.tsx:100
 msgid "Before creating a list, you must first verify your email."
-msgstr ""
+msgstr "Você deve verificar seu email antes de criar uma lista."
 
 #: src/view/com/composer/Composer.tsx:591
 msgid "Before creating a post, you must first verify your email."
-msgstr ""
+msgstr "Você deve verificar seu email antes de criar uma postagem."
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:1127]:
```suggestion
msgstr "Você deve verificar seu email antes de criar um post."
```

File: src/locale/locales/pt-BR/messages.po [hunk 10]
```diff
@@ -2443,7 +2443,7 @@ msgstr "Descartar rascunho?"
 
 #: src/view/com/composer/Composer.tsx:827
 msgid "Discard post?"
-msgstr ""
+msgstr "Descartar postagem?"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:2446]:
```suggestion
msgstr "Descartar post?"
```

File: src/locale/locales/pt-BR/messages.po [hunk 11]
```diff
@@ -8201,7 +8201,7 @@ msgstr "Este serviço de moderação está indisponível. Veja mais detalhes aba
 
 #: src/view/com/post-thread/PostThreadItem.tsx:836
 msgid "This post claims to have been created on <0>{0}</0>, but was first seen by Bluesky on <1>{1}</1>."
-msgstr ""
+msgstr "Esta postagem alega ter sido criada em <0>{0}</0>, mas ela apareceu no Bluesky em <1>{1}</1>."
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:8204]:
```suggestion
msgstr "Este post alega ter sido criada em <0>{0}</0>, mas ela apareceu no Bluesky em <1>{1}</1>."
```

File: src/locale/locales/pt-BR/messages.po [hunk 12]
```diff
@@ -9639,7 +9639,7 @@ msgstr "Sua postagem foi publicada"
 
 #: src/view/com/composer/Composer.tsx:459
 msgid "Your posts have been published"
-msgstr ""
+msgstr "Suas postagens foram publicadas"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:9642]:
```suggestion
msgstr "Seus posts foram publicados"
```

File: src/locale/locales/pt-BR/messages.po [hunk 13]
```diff
@@ -232,91 +232,91 @@ msgstr "{estimatedTimeMins, plural, one {minuto} other {minutos}}"
...
 
 #: src/view/com/notifications/FeedItem.tsx:326
 msgid "{firstAuthorLink} and <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}}</0> liked your custom feed"
-msgstr ""
+msgstr "{firstAuthorLink} e <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}}</0> curtiram seu feed personalizado"
 
 #: src/view/com/notifications/FeedItem.tsx:222
 msgid "{firstAuthorLink} and <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}}</0> liked your post"
-msgstr ""
+msgstr "{firstAuthorLink} e <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}}</0> curtiram sua postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:243]:
```suggestion
msgstr "{firstAuthorLink} e <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}}</0> curtiram seu post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 14]
```diff
@@ -232,91 +232,91 @@ msgstr "{estimatedTimeMins, plural, one {minuto} other {minutos}}"
...
 
 #: src/view/com/notifications/FeedItem.tsx:246
 msgid "{firstAuthorLink} and <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}}</0> reposted your post"
-msgstr ""
+msgstr "{firstAuthorLink} e <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}}</0> repostaram sua postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:247]:
```suggestion
msgstr "{firstAuthorLink} e <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}}</0> repostaram seu post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 15]
```diff
@@ -232,91 +232,91 @@ msgstr "{estimatedTimeMins, plural, one {minuto} other {minutos}}"
...
 
 #: src/view/com/notifications/FeedItem.tsx:319
 msgid "{firstAuthorName} and {additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}} liked your custom feed"
-msgstr ""
+msgstr "{firstAuthorName} e {additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}} curtiram seu feed personalizado"
 
 #: src/view/com/notifications/FeedItem.tsx:215
 msgid "{firstAuthorName} and {additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}} liked your post"
-msgstr ""
+msgstr "{firstAuthorName} e {additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}} curtiram sua postagem"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:287]:
```suggestion
msgstr "{firstAuthorName} e {additionalAuthorsCount, plural, one {{formattedAuthorsCount} outro} other {{formattedAuthorsCount} outros}} curtiram seu post"
```

File: src/locale/locales/pt-BR/messages.po [hunk 16]
```diff
@@ -913,27 +913,27 @@ msgstr "Idioma do aplicativo"
 
 #: src/screens/Settings/components/AddAppPasswordDialog.tsx:122
 msgid "App Password"
-msgstr ""
+msgstr "Senha de Aplicativo"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:916]:
```suggestion
msgstr "Senha do Aplicativo"
```

File: src/locale/locales/pt-BR/messages.po [hunk 17]
```diff
@@ -913,27 +913,27 @@ msgstr "Idioma do aplicativo"
...
 
 #: src/screens/Settings/AppPasswords.tsx:139
 msgid "App password deleted"
 msgstr "Senha de Aplicativo excluída"
 
 #: src/screens/Settings/components/AddAppPasswordDialog.tsx:84
 msgid "App password name must be unique"
-msgstr ""
+msgstr "O nome da senha de aplicativo deve ser único"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:924]:
```suggestion
msgstr "O nome da senha do aplicativo deve ser único"
```

File: src/locale/locales/pt-BR/messages.po [hunk 18]
```diff
@@ -913,27 +913,27 @@ msgstr "Idioma do aplicativo"
...
 
 #: src/screens/Settings/components/AddAppPasswordDialog.tsx:62
 msgid "App password names can only contain letters, numbers, spaces, dashes, and underscores"
-msgstr ""
+msgstr "O nome da senha de aplicativo só pode conter letras, números, travessões e underline"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:928]:
```suggestion
msgstr "O nome da senha do aplicativo só pode conter letras, números, travessões e underline"
```

File: src/locale/locales/pt-BR/messages.po [hunk 19]
```diff
@@ -913,27 +913,27 @@ msgstr "Idioma do aplicativo"
...
 
 #: src/view/com/modals/AddAppPasswords.tsx:138
 #~ msgid "App Password names can only contain letters, numbers, spaces, dashes, and underscores."
 #~ msgstr "O nome da Senha de Aplicativo só pode conter letras, números, travessões e underline."
 
 #: src/screens/Settings/components/AddAppPasswordDialog.tsx:80
 msgid "App password names must be at least 4 characters long"
-msgstr ""
+msgstr "O nome da senha de aplicativo deve ter pelo menos 4 caracteres"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:936]:
```suggestion
msgstr "O nome da senha do aplicativo deve ter pelo menos 4 caracteres"
```

File: src/locale/locales/pt-BR/messages.po [hunk 20]
```diff
@@ -946,7 +946,7 @@ msgstr ""
 #: src/screens/Settings/PrivacyAndSecuritySettings.tsx:56
 #: src/screens/Settings/PrivacyAndSecuritySettings.tsx:59
 msgid "App passwords"
-msgstr ""
+msgstr "Senhas de aplicativo"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:949]:
```suggestion
msgstr "Senhas do aplicativo"
```

File: src/locale/locales/pt-BR/messages.po [hunk 21]
```diff
@@ -1000,20 +1000,20 @@ msgstr "Utilizar feeds recomendados"
...
+msgstr "Post arquivado"
 
 #: src/screens/StarterPack/StarterPackScreen.tsx:610
 #~ msgid "Are you sure you want delete this starter pack?"
 #~ msgstr "Tem certeza de que deseja excluir este pacote inicial?"
 
 #: src/screens/Settings/AppPasswords.tsx:201
 msgid "Are you sure you want to delete the app password \"{0}\"?"
-msgstr ""
+msgstr "Tem certeza de que deseja excluir a senha de aplicativo \"{0}\"?"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:1016]:
```suggestion
msgstr "Tem certeza de que deseja excluir a senha do aplicativo \"{0}\"?"
```

File: src/locale/locales/pt-BR/messages.po [hunk 22]
```diff
@@ -2004,11 +2004,11 @@ msgstr "Copiar"
 
 #: src/screens/Settings/components/AddAppPasswordDialog.tsx:196
 msgid "Copy App Password"
-msgstr ""
+msgstr "Copiar Senha de Aplicativo"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:2007]:
```suggestion
msgstr "Copiar Senha do Aplicativo"
```

File: src/locale/locales/pt-BR/messages.po [hunk 23]
```diff
@@ -7200,7 +7200,7 @@ msgstr "Mostrar respostas ocultas"
 
 #: src/view/com/post-thread/PostThreadItem.tsx:796
 msgid "Show information about when this post was created"
-msgstr ""
+msgstr "Mostrar informação sobre quando esta postagem foi criada"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:7203]:
```suggestion
msgstr "Mostrar informação sobre quando este post foi criado"
```

File: src/locale/locales/pt-BR/messages.po [hunk 24]
```diff
@@ -7978,7 +7978,7 @@ msgstr "Tivemos um problema ao carregar esta lista. Toque aqui para tentar de no
 
 #: src/screens/Settings/AppPasswords.tsx:52
 msgid "There was an issue fetching your app passwords"
-msgstr ""
+msgstr "Houve um problema ao recuperar suas senhas de aplicativo"
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:7981]:
```suggestion
msgstr "Houve um problema ao recuperar suas senhas do aplicativo"
```

File: src/locale/locales/pt-BR/messages.po (hunk 11 above)
Comment (luan-u) [inline src/locale/locales/pt-BR/messages.po:8204]:
```suggestion
msgstr "Este post alega ter sido criado em <0>{0}</0>, mas ele apareceu no Bluesky em <1>{1}</1>.""
```

File: src/locale/locales/pt-BR/messages.po [hunk 25]
```diff
@@ -8145,7 +8145,7 @@ msgstr "Este feed não funciona mais. Estamos te mostrando o conteúdo do <0>Dis
 
 #: src/screens/Settings/components/ChangeHandleDialog.tsx:559
 msgid "This handle is reserved. Please try a different one."
-msgstr ""
+msgstr "Este usuário está reservado. Por favor tente um usuário diferente."
```
Comment (pfrazee) [inline src/locale/locales/pt-BR/messages.po:8148]:
```suggestion
msgstr "Este nome de usuário está reservado. Por favor tente um nome de usuário diferente."
```

**Review by pfrazee — APPROVED** [review]
Awesome tysm! A couple of tweaks

**Pull request merged**